doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -22,7 +22,9 @@ \subsubsection{Phase diagram}
 
 \par The material model is then set up to mimic mantle phase transitions at 410, 520, and 660 km.% setup of mantle transitions
 Details of phase transitions are taken from \cite{billen2018decoupling}.
-A trick of assigning the values of density to the field of thermal capacity is used here [MIB: How is this done? can you be more explicit]. % trick
+A trick of assigning the values of density to the field of thermal capacity is used here. % [MIB: How is this done? can you be more explicit, addressed]. % trick
```

Comment:
Can you change this to:  "A trick is needed to make this work: I assign the values of the density to the field of thermal capacity in the input file."